Documentation changes from 4.14 release for docs migration (#1590)
apache/cassandra-java-driver: modules/ROOT/pages/core/integration.adoc

Our xref:core/dse/geotypes.adoc[geospatial types] implementation is based on the https://github.com/Esri/geometry-api-java[Esri Geometry API].

For driver versions >= 4.4.0 and < 4.14.0 Esri is declared as a required dependency,
although the driver can operate normally without it. If you don't use geospatial types

> anywhere in your application you can exclude the dependency:

Reviewer: ```suggestion
anywhere in your application, you can exclude the dependency:
```

</dependency>
----

Starting with driver 4.14.0 Esri has been changed to an optional dependency.  You no longer have to
explicitly exclude the dependency if it's not used, but if you do wish to make use of the Esri

> library you must now explicitly specify it as a dependency :

Reviewer: ```suggestion
Starting with driver 4.14.0, Esri has been changed to an optional dependency.  If the dependency is not used, you 
no longer have to explicitly exclude the dependency. If you want to use the Esri
library, explicitly specify it as a dependency:
```

[source,xml]
----
<dependency>
  <groupId>com.esri.geometry</groupId>
  <artifactId>esri-geometry-api</artifactId>
  <version>${esri.version}</version>
</dependency>
----

In the dependency specification above you should use any 1.2.x version of Esri (we recommend
1.2.1).  These versions are older than the current 2.x versions of the library but they are

> guaranteed to be fully compatible with DSE.

Reviewer: ```suggestion
In the dependency specification above, use any 1.2.x version of Esri. DataStax recommends
1.2.1. These versions are older than the current 2.x versions of the library, but they are
guaranteed to be fully compatible with DSE.
```
